pkg/pillar/cmd/domainmgr/domainmgr.go

```diff
@@ -792,6 +813,27 @@ func unpublishCipherBlockStatus(ctx *domainContext, key string) {
 	pub.Unpublish(key)
 }
 
+func createLocalAppActiveFile(appUUID string) {
+	// Construct the file path as "<LocalActiveAppConfigDir>/<appUUID>.json".
+	filePath := filepath.Join(types.LocalActiveAppConfigDir, appUUID+".json")
+
+	// Ensure that the base directory exists.
+	if err := os.MkdirAll(types.LocalActiveAppConfigDir, 0755); err != nil {
```

Comment:
```suggestion
	if err := os.MkdirAll(types.LocalActiveAppConfigDir, 0700); err != nil {
```
I guess other users do not need access to it, do they?